Example/Pods/Local Podspecs/ILGDynamicObjC.podspec.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "ILGDynamicObjC",
+  "version": "0.1.1",
```

Comment:
I'd have expected this to be `0.2.0`—did you `pod install` or `pod update ILGDynamicObjC` in the Example project after changing the podspec file?